spec/spec_helper.rb

```diff
@@ -4,15 +4,10 @@
 require 'simplecov'
 require 'coveralls'
 
-# if ENV['TRAVIS'] == 'true'
-#   # don't generate local report as it is inaccessible on travis-ci, which is why coveralls is being used.
-#   SimpleCov.formatter = Coveralls::SimpleCov::Formatter
-# else
   SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new(
```

Comment:
Does this provide the same functionality?

```suggestion
if ENV['CI'] == 'true'
  # don't generate local report as it is inaccessible on travis-ci, which is why coveralls is being used.
  SimpleCov.formatter = Coveralls::SimpleCov::Formatter
else
  SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new(
```

https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/

Reply:
Or rather, what did this old code do, and do we need to make any changes here?

Reply:
It was originally commented out, so I just removed it as it was referring to Travis.

Reply:
Can we delete the `coveralls` references elsewhere now? :+1:
Such as the gem requires and above in the spec helper